ajax/libs/Peppermint/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Peppermint",
+  "version": "1.4.0",
+  "license": "MIT",
+  "filename": "peppermint.min.js",
+  "description": "Responsive touch-enabled slider. Tiny, supports mouse and all sorts of touch, library agnostic.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wilddeer/Peppermint.git"
+  },
+  "author": {
+    "name": "Oleg Korsunsky",
+    "url": "http://wd.dizaina.net/"
+  },
+  "keywords": [
+    "touch",
+    "touch-events",
+    "pointer-events",
+    "mouse-events",
+    "slider",
+    "carousel",
+    "gallery",
+    "vanilla-js",
+    "framework-agnostic",
+    "library-agnostic"
+  ],
+  "homepage": "http://wd.dizaina.net/en/scripts/peppermint/",
```

Comment:
This change should be in the first commit.
Please delete this line in the second commit, and use `git rebase -i HEAD~2` to `edit` first commit.